bin/server.js

```diff
@@ -14,12 +14,24 @@ var url_parser = require('url');
 
 var name = process.env['kong-dashboard-name'];
 var pass = process.env['kong-dashboard-pass'];
+var url = process.env['kong-dashboard-url'];
```

Comment:
Environment variable name is confusing because it lets think that it is the url of Kong Dashboard although it contains the url of Kong. I would suggest `kong-dashboard-kong-url`.

Reply:
or maybe kong-dashboard-node-url